packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -512,6 +519,9 @@ class PopupMenuButton<T> extends StatefulWidget {
   /// The widget below this widget in the tree.
   final Widget child;
 
+  /// The icon used for this button.
+  final Icon icon;
+
```

Comment:
I described that [here with the class-level docs](https://github.com/flutter/flutter/pull/10230/commits/30b085b6ca19974259b878bf0f04c22e7839de49#diff-437fce7f7d560afa95e469f92c2c8808R473).